### DISCUSSION TITLE
本地部署之后手机扫码显示当前账户无法使用该服务

### DISCUSSION THREAD

**Discussion by Joshmomel** [zhayujie/chatgpt-on-wechat#2017] [Q&A]
你好, 

我在本地server部署, 运行正常, 但是用我的手机微信扫码之后显示: 前账户无法使用该服务

运行给出的4个链接都试了, 都不能登录, 请问该怎么解决

**Comment by dorothy-in-ds** [comment]
你可以登录网页版微信吗？我不可以登录网页版微信，手机扫本地部署产生的二维码也有同样的问题。人在境外，怀疑是这个原因。
